python-dictionaries/dict_solution.py

```diff
@@ -0,0 +1,17 @@
+def deci_to_bi(number):
+
+  # List of keys (decimals)
+  deci_list = []
+  for i in range(number):
+    deci_list.append(i)
+
+  # List of values (binaries)
+  bi_list = []
+  for i in range(number):
+    bi_list.append("{:04b}".format(i))

```

Comment:
This is technically correct, but the goal of the assignment is to get practice writing out algorithms. See if you can come up with code to manually convert decimal to binary (good use of formatting though!)